x-pack/test/functional/es_archives/cases/migrations/7.13.2/mappings.json

```diff
@@ -76,7 +76,6 @@
           "security-rule": "8ae39a88fc70af3375b7050e8d8d5cc7",
           "security-solution-signals-migration": "72761fd374ca11122ac8025a92b84fca",
           "siem-detection-engine-rule-actions": "6569b288c169539db10cb262bf79de18",
-          "siem-detection-engine-rule-status": "ae783f41c6937db6b7a2ef5c93a9e9b0",
```

Comment:
Just curious about these `7.x` migration mappings -- are these not needed from a historical perspective since the `siem-detection-engine-rule-status` SO was present in `7.13.2` in this example. Or are these specific `7.13.2` mappings used for migrating to `main` and thus not needed since `siem-detection-engine-rule-status` migrations are handled in other tests?

Reply:
Well, that's a good question. I don't know 🙂 

First, I'm not sure I understand why mappings of `siem-detection-engine-rule-status` were used in integration tests for `cases/migrations`. Tests that check cases migrations should only use the mappings of SOs related to cases. So removing this (since it doesn't break those tests) looks like an improvement to me. 

Respectively, I'd expect mappings of `siem-detection-engine-rule-status` to be used only in the tests that check this SO migrations.

Reply:
@cnasikas could you please review this change?

Reply:
cc @jonathan-buttner

Reply:
@cnasikas I see, thank you for looking into this. In this case, if the file is auto-generated, I'd prefer to keep it as is, so I'm gonna revert changes in those es_archives in this PR.

Reply:
Thanks!

Reply:
Yep I agree, if we could remove the change that'd be great!

Reply:
Okie, so it's now addressed. Thank you @spong @cnasikas @jonathan-buttner!